### PR TITLE
Fix/tec 3009 start end time same

### PR DESCRIPTION
**Pull request by paulmskim**
[TEC-3009]

Screencast: https://drive.google.com/open?id=1amuxu6xMDPKLyivpzudXzXHTQP6Eg2oo

[TEC-3009]: https://moderntribe.atlassian.net/browse/TEC-3009

### CODE REVIEW

File: readme.txt [hunk 1]
```diff
@@ -6,6 +6,7 @@
 
 * Fix - Fix style overrides for new view shortcodes for Genesis theme. [ECP-316]
 * Fix - Fix style overrides for new view shortcodes for Enfold theme. [ECP-315]
+* Tweak - Update adjust start function in moment utils to allow start and end time to be the same. [TEC-3009]
```
Comment (sc0ttkclark) [inline readme.txt]:
```suggestion
* Tweak - Update `adjustStart` function in moment.js to allow start and end time to be the same. [TEC-3009]
```

File: readme.txt [hunk 2]
```diff
@@ -6,6 +6,7 @@
 
 * Fix - Fix style overrides for new view shortcodes for Genesis theme. [ECP-316]
 * Fix - Fix style overrides for new view shortcodes for Enfold theme. [ECP-315]
+* Tweak - Update `adjustStart` function in moment.js to allow start and end time to be the same. [TEC-3009]
```
Comment (sc0ttkclark) [inline readme.txt]:
```suggestion
* Tweak - Update `adjustStart()` function in moment.js to allow start and end time to be the same. [TEC-3009]
```

File: readme.txt [hunk 3]
```diff
@@ -6,6 +6,7 @@
 
 * Fix - Fix style overrides for new view shortcodes for Genesis theme. [ECP-316]
 * Fix - Fix style overrides for new view shortcodes for Enfold theme. [ECP-315]
+* Tweak - Update `adjustStart()` function in moment.js to allow start and end time to be the same. [TEC-3009]
```
Comment (paulmskim) [inline readme.txt]: @sc0ttkclark the `adjustStart` function is actually from our own moment utils, not from `moment.js`. So how I had written it previously is correct.

Comment (paulmskim) [reply readme.txt]: merging to unblock https://github.com/moderntribe/the-events-calendar/pull/2998. see https://github.com/moderntribe/tribe-common/pull/1260

**Pull request merged**